cli/cli_app.go

```diff
@@ -248,6 +251,21 @@ func runTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 		}
 	}
 
+	if shouldPrintTerragruntInfo(terragruntOptions) {
+		terragruntOptions.Logger.Printf("\n\n"+
+			"terragrunt workingdir: %s\n"+
+			"terragrunt config path: %s\n"+
+			"terraform binary: %s\n"+
+			"terraform command: %s\n"+
+			"DownloadDir: %s\n",
+			terragruntOptions.WorkingDir,
+			terragruntOptions.TerragruntConfigPath,
+			terragruntOptions.TerraformPath,
+			terragruntOptions.TerraformCommand,
+			terragruntOptions.DownloadDir)
```

Comment:
Should this be outputted in JSON format for easier machine consumption?

Reply:
I did see your earlier suggestions to output in JSON format, but I didn't see any instances where `terragrunt` performs any `write()` calls to `stdout`.

I've dug into things and now think `terragruntOptions.Writer` is the correct Writer to use, and have updated the PR to emit JSON on stdout.